sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts

```diff
@@ -230,12 +230,27 @@ export class ScholasticStandingInstitutionsController extends BaseController {
     // Validate the text models.
     const textValidations =
       this.applicationWithdrawalImportTextService.validateTextModels(
-        textModels,
+        withdrawalFileData.applicationWithdrawalModels,
```

Comment:
With the renaming of the class, my intention was to have the properties simplified as, for instance, `withdrawalFileData.records`, since we already have `withdrawalFileData.header`. Does it make sense?